egs/librispeech/ASR/pruned_transducer_stateless5/ConvRNN-T.py

```diff
@@ -0,0 +1,1029 @@
+#!/usr/bin/env python3
```

Comment:
Please create a new directory, eg, pruned_transducer_stateless7, and add training script for it.